tests/testBackground.py

```diff
@@ -218,7 +183,8 @@ def testRamp(self):
         bctrl.getStatisticsControl().setNumIter(1)
         backobj = afwMath.cast_BackgroundMI(afwMath.makeBackground(rampimg, bctrl))
 
-        print(rampimg.getArray())
+        if display:
+            print(rampimg.getArray())
```

Comment:
Should I change `display` to `debugMode` or something similar? 

Alternatively, I could create a ticket to remove print functions in tests and replace them with lsst.log (once logging has been completed). This would eliminate the need for a debug variable and is the proper way to show debugging info.

For debug code that uses ds9, the module could check whether or not the logging level is <= DEBUG and only use ds9 if this is True.